lib/ruby/truffle/truffle/bundler-workarounds.rb

```diff
@@ -0,0 +1,516 @@
+$quiet_workaround = false
+
+workaround_header = <<~HEREDOC
```

Comment:
Squiggly heredoc is a very new feature, maybe it's better to stick to `<<-` here, since anyway the indentation is not needed.